Rudder.podspec

```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
 
   s.homepage         = "https://github.com/rudderlabs/rudder-sdk-ios"
   s.license          = { :type => "Apache", :file => "LICENSE" }
-  s.author           = { "RudderStack" => "arnab@rudderlabs.com" }
+  s.author           = { "RudderStack" => "sdk-accounts@rudderlabs.com" }
```

Comment:
Should this be `sdk-accounts@rudderstack.com`?

Reply:
Yes. Corrected.